acceptance/analyzer_test.go

```diff
@@ -474,9 +474,8 @@ func testAnalyzerBuilder(apiString string) func(t *testing.T, when spec.G, it sp
 								)
 							})
 
-							it.After(func() {
-								h.DockerImageRemove(t, authRegCacheImage)
-							})
+							// Don't attempt to remove the image, as it's stored in the test registry, which is ephemeral.
```

Comment:
Does it worth adding to all of these comments a reference to `defer registry.Stop(t)` for readability purpose?
Also, not related to this PR, but I think that the name of the `Stop` function isn't accurate since `ContainerKill` and `ContainerRemove` are being called there.

Reply:
I didn't mean that we should mention `defer registry.Stop` in the comment since we might change the function name.
I don't have a specific idea so let's leave the comment as is for now and we can always change it in a future PR if we think about a better comment.